src/data/faq.json

```diff
@@ -87,6 +87,117 @@
+                        ]
+                    },
+                    {
+                        "title": "Are warnings from a rapid antigen or PCR test differentiated?",
+                        "anchor": "rat_warning_not_differentiated",
+                        "active": true,
+                        "textblock": [
+                            "Contact persons receive a warning in their Corona warning app about an encounter on a day with low (green tile) or increased risk (red tile). Whether the warning was based on a rapid test or on a PCR test is not displayed in the app."
+                        ]
+                    },
+                    {
+                        "title": "EFGS integration? Are positive results shared in other countries?",
+                        "anchor": "rat_efgs",
+                        "active": true,
+                        "textblock": [
+                            "Warnings based on a rapid test result are not shared with other EU countries via the EFGS (European Federation Gateway Service), including Switzerland, as the EFGS terms of use do not provide for rapid test warnings to be shared."
+                        ]
+                    },
+                    {
+                        "title": "What is the difference between PCR tests, rapid antigen tests and self-tests?",
+                        "anchor": "rat_differences_pcr_rat",
+                        "active": true,
+                        "textblock": [
+                            "PCR tests are the 'gold standard' among corona tests. Sampling is performed by medical personnel - evaluation by laboratories.",
+                            "Rapid antigen tests: Have their name because the result is available quickly. They can only be performed by trained personnel. Similar to the PCR test, a nasal or throat swab is taken. However, in contrast to the PCR test, the evaluation takes place directly on site. Since March 8, 2021, everyone is entitled to at least one rapid test per week.",
+                            "Self-tests: Have their name because anyone can do these tests themselves, for example at home. The self-tests are intended for use by private individuals. For this purpose, sample collection and evaluation must be appropriately simple. The test can be performed with a nasal swab or saliva, for example. The Federal Institute for Drugs and Medical Devices, together with the Paul Ehrlich Institute, checks the quality and significance of the tests.",
+                            "Rapid and self-tests have a higher error rate compared to PCR tests. Therefore, after each positive rapid and self-test, a PCR test should always be done for confirmation."
+                        ]
+                    },
+                    {
+                        "title": "Can I save/manage multiple rapid antigen tests?",

```

Comment:
The German FAQ follows with the entry "rat_rat_tan_hotline" and the the "rat_multiple_rat".

Reply:
removed one rat, hahah ;-)

Reply:
there is no problem with multiple rats.... ;)

rat_tan_hotline article is missing in EN

Reply:
thanks for the hint, that something wrong here ...
its confusing with removing one rat, and still multiple_rats(s) is ok.

Reply:
should be fixed now.